test/verify/files/mock-faf-server.py

```diff
@@ -45,10 +45,11 @@ def do_POST(self):
                     'reporter': 'Bugzilla'
                 }
             ],
-            'result': False
+            'result': Handler.known.next()
         }
         json.dump(response, self.wfile, indent=2)
 
 PORT = 12345
+Handler.known = [True, False].__iter__()
```

Comment:
This would need to be adjusted for testing more cases. Simple enough, but maybe add a comment about it to `def testAbrtReport()`? Something like

    # Adjust Handler.known in test/verify/files/mock-faf-server.py for the expected succession of known/unknown

?